src/main/java/com/google/api/codegen/gapic/MainGapicProviderFactory.java

```diff
@@ -235,10 +220,27 @@
               .setCodePathMapper(pythonPathMapper)
               .build();
 
+      if (id.equals(PYTHON)) {
+        System.out.println("stopping here");
```

Comment:
That's why I mark this kind of stuff with FIXME, so it's obvious it needs to go before approval.